src/meta/meta_bulk_load_service.cpp

```diff
@@ -1019,6 +1019,14 @@ void bulk_load_service::on_partition_ingestion_reply(error_code err,
                                                      const std::string &app_name,
                                                      const gpid &pid)
 {
+    if (err == ERR_NO_NEED_OPERATE) {
```

Comment:
`ERR_NO_NEED_OPERATE` seems to be not clear, how about `ERR_OPERATION_REPEATED`

Reply:
`ERR_NO_NEED_OPERATE` is a existed error code. If I use `ERR_OPERATION_REPEATED`, I should define a new one.